pycloudlib/oci/cloud.py

```diff
@@ -108,13 +108,14 @@ def __init__(
         self.compute_client = oci.core.ComputeClient(self.oci_config)
         self.network_client = oci.core.VirtualNetworkClient(self.oci_config)
 
-    def delete_image(self, image_id):
+    def delete_image(self, image_id, **kwargs):
```

Comment:
I wonder if we should propagate the `**kwargs` definition to the base class and every child class.

By doing so, we would allow other instances of other child classes to ignore extra `kwargs` without breaking. For example, the following call would raise if `ec2` is not an instance of `oci.cloud.Cloud`:

```python
ec2.delete_image("image-id-asdf", other_param="qwer")
```